test/utils/createElement.ts

```diff
@@ -0,0 +1,30 @@
+import { render, type TemplateResult } from "lit-html"
+
+/**
+ * Converts a lit-html template result into a DOM element
+ * @param template - The template result from lit-html's html function
+ * @returns The parsed DOM element
+ * @throws Error if the template does not produce a valid element
+ */
+export function createElement(template: TemplateResult): HTMLElement {
```

Comment:
add a type parameter E extends HTMLElement and use it in place of type assertions